pkg/cmd/gist/view/view.go

```diff
@@ -160,3 +180,47 @@ func viewRun(opts *ViewOptions) error {
 
 	return nil
 }
+
+func promptGists(client *http.Client) (gistID string, err error) {
+	gists, err := list.ListGists(client, ghinstance.OverridableDefault(), 10, "all")
+	if err != nil {
+		return "", err
+	}
+
+	if len(gists) == 0 {
+		return "", nil
+	}
+
+	var opts []string
+	var result int
+	var gistIDs = make([]string, len(gists))
+
+	for i, gist := range gists {
+		gistIDs[i] = gist.ID
+		description := gist.Description
+		if description == "" {
+			for filename := range gist.Files {
+				if !strings.HasPrefix(filename, "gistfile") {
+					description = filename
+					break
+				}
+			}
+		}
+		gistTime := utils.FuzzyAgo(time.Since(gist.UpdatedAt))
+		opt := fmt.Sprintf("%s (%s)", text.ReplaceExcessiveWhitespace(description), gistTime)
```

Comment:
It's still possible for description to end up blank which looks weird in the prompt; how about we try setting description to something like `"<no description>"` if it still ends up empty at this point?

Reply:
### Updated `gist view`
![Screen Shot 2021-02-24 at 3 02 13 PM](https://user-images.githubusercontent.com/10766871/108961720-b782f000-76b2-11eb-9659-0cca67a50336.png)
I show the (default?) filename `gistfile1.txt` which is not provided by user when creating gist.

---

### Current `gist list`
Also, I found current `gist list` won't show anything if both filename and description are empty. Is it desired result?
![Screen Shot 2021-02-24 at 3 03 20 PM](https://user-images.githubusercontent.com/10766871/108962431-b3a39d80-76b3-11eb-833b-981a64fb5f8e.png)

---

### GitHub UI gists
GitHub UI gists listing page uses `gist:[node_name]` when there's no filename:
![Screen Shot 2021-02-24 at 3 27 42 PM](https://user-images.githubusercontent.com/10766871/108963299-e1d5ad00-76b4-11eb-955e-b3910394ace4.png)

Reply:
My concern was the blank at the start of the line; just using `gistfile` is fine for now.